lib/system.nim

```diff
@@ -4210,6 +4210,23 @@ when hasAlloc or defined(nimscript):
       x[j+i] = item[j]
       inc(j)
 
+  func insert*(x: var string, c: char, i = 0.Natural) =
+    ## Inserts character `c` into `x` at position `i`
+    ##
+    ## .. code-block:: Nim
+    ##   var a = "hllo"
+    ##   a.insert('e', 1) # a <- "Hello"
+    var xl = x.len
+    setLen(x, xl+1)
+    var j = xl-1
```

Comment:
Please use descriptive variable names, also the white spaces between the operators makes it nicer to read.
:slightly_smiling_face:

Reply:
True, but I just copied the implementation from insert(string, string), so that is where this came from.